packages/validators/typescript/src/TypeScriptValidator.js

```diff
@@ -129,6 +124,7 @@ async function tryCreateLanguageService(
       langServiceCache[config.configHash] = {
         configHost,
         host,
+        // $FlowFixMe - complains about methods being readonly...
```

Comment:
nit: this can be narrowed to the specific violation, e.g. `// $FlowFixMe[violation-name]`

Reply:
Ah TIL! It's actually two violations but I added them both